udata_datalake_service/background_tasks.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+import tempfile
+
+import boto3
+from celery import Celery
+import requests
+from botocore.client import Config
+from botocore.exceptions import ClientError
+from dotenv import load_dotenv
+
+from udata_datalake_service.producer import produce
+
+load_dotenv()
+
+BROKER_URL = os.environ.get("BROKER_URL", "redis://localhost:6380/0")
+MINIO_FOLDER = os.environ.get("MINIO_FOLDER", "folder")
+celery = Celery('tasks', broker=BROKER_URL)
+
+
+def download_resource(url):
+    tmp_file = tempfile.NamedTemporaryFile(delete=False)
+    with requests.get(url, stream=True) as r:
+        r.raise_for_status()
+        for chunk in r.iter_content(chunk_size=1024):
```

Comment:
Do we add a maximum size here as a parameter ? Maybe we do not want to download in our catalog resource more than XX Go ?